### ISSUE TITLE
Custom shortcut to launch guake not working after upgrade from 0.4.4 to 0.8.3 (shift+alt+t)

### ISSUE THREAD

**Issue by ElijahLynn** [Guake/guake#697]
I had a custom shortcut of `shift+alt+t` to launch Guake in 0.4.4. And just upgraded to 0.8.3 and the shortcut doesn't work anymore. 

I am on Ubuntu 14.04 and using the webupd PPA. I completely uninstalled with a `apt-get purge guake` too but no luck. 

I was however to get a ctrl+` shortcut working though, but really want my old shortcut back.

**Comment by Stibbons** [comment]
launch or display? we have changed to key binder library this may be the cause. does it work with other shortcut?

**Comment by ElijahLynn** [comment]
Display, once it is already launched I cannot use and shift+alt+t combination. I can use ctrl + ` though.

**Comment by ElijahLynn** [comment]
Interesting, I also cannot paste or copy with ctrl + shift +c/v anymore. I didn't relate this but I think they are related.

**Comment by ElijahLynn** [comment]
I just jumped back to 0.7.2 and the ctrl+shift+t keyboard shortcut is working now. Still no copy/paste though.

**Comment by Stibbons** [comment]
Indeed, Ctrl+shift+[any letter] doesn't seem to work, but Ctrl+Shift+F12 works. Weird

**Comment by ElijahLynn** [comment]
Thanks for confirming this.

**Comment by ElijahLynn** [comment]
Possibly related to #510

**Comment by ElijahLynn** [comment]
Possibly related to #560

**Comment by Stibbons** [comment]
Thanks

**Comment by ElijahLynn** [comment]
So, in #560 I see that one of the issues is there is no unbind_all() anymore and we need to specifically call unbind. There is one place where we unbind now => https://github.com/Guake/guake/pull/560/files#diff-455aac7e7f11bc78c3a502a3e136925eR1009. I don't actually have a "~/.gconf/apps/guake/keybindings/global/show_hide". Not sure if gconf_path means it is actually looking there though. Looks like I should be able to get the dev version installed again and step debug this soon.

**Comment by mouseratti** [comment]
Friends, I have a similar problem. After upgrading from *0.8.1* to *0.8.6*, hotkeys for new tab(Ctrl-E in my configuration) and close tab(Ctrl-w) are not working anymore. I rolled back to version 0.8.1 and these functions works as expected now. I tested it on versions 0.8.2, 0.8.3 0.8.4 too - hotkeys not work on these versions.

**Comment by jbruni** [comment]
Any news? I just want to hit F12 and have Guake available, as I'm used to...
